src/errno.rs

```diff
@@ -0,0 +1,24 @@
+extern crate libc;
+use std::ffi::CString;
+use std::os::raw::c_char;
+use libc::c_int;
+use fileops::open;
+
+pub fn strerrer(errno: c_int) -> String {
+  unsafe {
+    let c_err_str = libc::strerror(errno);
+    CString::from_raw(c_err_str).into_string()
+      .unwrap_or(String::from("Unknown errno"))
```

Comment:
Would it be helpful to include the errno here?

```rust
format!("Unknown errno: {}", c_err_str)
```

dunno if c_err_str impls Display, needed for format! (and println!)